java/code/src/com/suse/manager/reactor/messaging/RegisterMinionEventMessageAction.java

```diff
@@ -299,6 +291,8 @@ else if (!server.getOrg().equals(org)) {
             server.setServerArch(
                     ServerFactory.lookupServerArchByLabel(osarch + "-redhat-linux"));
 
+
+
```

Comment:
Two empty lines not needed

Reply:
Already fixed by last commit: https://github.com/uyuni-project/uyuni/pull/59/commits/75ca4d31583956bff0bc24a72c491d8a90aa3fff#diff-a112668bac573a2c5fb5981f604ffb99L294